src/components/dashboard/components/DictionariesPaginations.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+const Paginations = props => (
+  <Fragment>
+    {props.totalDictionaries > 0 && (
+      <div className="container">
+        <div className="row pagination-container">
+          <div className="col-12 text-center">
+            <div className="row">
+              <div className="col-11 text-right">
+                <i>
+                  Showing {props.firstDictionaryIndex}-{props.dictionaries} of{' '}
+                  {props.totalDictionaries} dictionaries
+                </i>
+                <span className="paginate-controllers">
+                  {props.currentPage > 1 ? (
+                    <i
+                      className="fas fa-angle-double-left prev"
+                      role="presentation"
+                      id={props.currentPage - 1}
+                      onClick={props.handleClick}
+                    />
+                  ) : (
+                    <i className="fas fa-angle-double-left disabled" role="presentation" />
+                  )}
+                  <span className="badge badge-light">
+                    <h6>{props.currentPage}</h6>
+                  </span>
+                  {props.currentPage < props.lastPage ? (
+                    <i
+                      className="fas fa-angle-double-right nxt"
+                      role="presentation"
+                      id={props.currentPage + 1}
+                      onClick={props.handleClick}
+                    />
+                  ) : (
+                    <i className="fas fa-angle-double-right disabled" role="presentation" />
+                  )}
+                </span>
+              </div>
+              <div className="col-1 text-left">
+                <select
+                  className="form-inline input-sm"
+                  onChange={e => props.onChangeLimit(e.target.value)}
```

Comment:
`changeLimit` or `handleChangeLimit` seems a more descriptive function name.

Reply:
@dondrzzy `handleChangeLimit` is a good name for this for this method. Thanks.